include/procfile.bash

```diff
@@ -33,7 +33,10 @@ procfile-types() {
 		local default_types
 		default_types="$(cat $app_path/.release | yaml-keys default_process_types | xargs echo)"
 		[[ "$default_types" ]] && \
-			echo "Default process types for $selected_name -> ${default_types// /, }"
+			echo "Default types for $selected_name -> ${default_types// /, }"
+			for type in $default_types; do
+				echo "$type: $(cat $app_path/.release | yaml-get default_process_types $type)" >> "$app_path/Procfile"
+			done
```

Comment:
is this indented correctly? it's always going run, is that desired? i suppose this is the problem with `&&` conditional statements.